packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -147,6 +147,7 @@ class _CupertinoSwitchState extends State<CupertinoSwitch> with TickerProviderSt
         onChanged: widget.onChanged,
         vsync: this,
         dragStartBehavior: widget.dragStartBehavior,
+        enableOnOffLabels: SemanticsBinding.instance.accessibilityFeatures.onOffSwitchLabels,
```

Comment:
Rather than use this directly, it's better to add this field to `MediaQueryData`, populate the data there from the binding (take a look at how we do this for `accessibleNavigation`) then use the field from `MediaQueryData`. here.